### PR TITLE
Issue 1051 models managing

### LINKED ISSUE #1051: Docs needed: Models > Managing
This issue spawned from #1008 .
This new file needs to be populated: models-managing.md .
Be sure to include `juju switch` usage.

### PR DESCRIPTION
**Pull request by pmatulis**
Fixes #1051

### CODE REVIEW

File: src/navigation.tpl [hunk 1]
```diff
@@ -19,10 +19,9 @@
         </li>
         <li class="section"><a class="header" href="models.html">Models</a>
             <ul class="sub">
-                <li><a href="models-adding.html">Adding</a></li>
-                <li><a href="models-managing.html">Managing</a></li>
-                <li><a href="models-configuring.html">Additional configuration</a></li>
+                <li><a href="models-config.html">Configuring</a></li>
                 <li><a href="models-upgrade.html">Upgrading</a></li>
+                <li><a href="models-ha.html">Juju HA</a></li>
```
Comment (evilnick) [inline src/navigation.tpl]: Isn't HA only applicable to the controller?

File: src/en/models.md [hunk 2]
```diff
@@ -1,30 +1,165 @@
...
+   
+   Juju software is upgraded at the model level. This affects the Juju agents
+   running on every machine Juju creates. This upgrade process does not pertain
+   to the Juju software package installed on a client system.
+
+   This is a complex subject. See [Juju upgrades](./models-upgrade.html).
+
+
+
+^# Set up HA for a model
```
Comment (evilnick) [inline src/en/models.md]: as mentioned elsewhere, isn't HA applied at the controller level?

File: src/en/models.md [hunk 3]
```diff
@@ -1,30 +1,165 @@
...
+   
+
+^# Grant and revoke user access
+   
+   This topic is treated in
+   [Users and models](./users-models.html#models-and-user-access).
+
+
+
+^# Configure a model
```
Comment (evilnick) [inline src/en/models.md]: I would move this further up the list

**Comment by evilnick** [comment]
comments made inline.

Looks good, but beware of foldouts - because of that bug in the way they are processed, i'm not sure your examples will be rendered properly.

Otherwise :+1:, :dolphin:

File: src/navigation.tpl (hunk 1 above)
Comment (pmatulis) [inline src/navigation.tpl]: Yeah, you're right. It is only HA for the model in an abstract, architectural sense.

File: src/en/models.md (hunk 2 above)
Comment (pmatulis) [inline src/en/models.md]: Yeah.

File: src/en/models.md (hunk 3 above)
Comment (pmatulis) [inline src/en/models.md]: OK

**Comment by pmatulis** [comment]
Re rendering, let's see what happens. I'll follow up accordingly.

Since this organization is acceptable we can go on to do the same for section 'Controllers'.

**Pull request merged**